trafilatura/metadata.py

```diff
@@ -46,13 +46,26 @@ def normalize_json(inputstring):
     return trim(inputstring)
 
 
+def normalize_authors(author):
+    # fix to code with unicode
+    if '\\u' in author:
+        author = author.encode().decode('unicode_escape')
+    # simple filters for German and English
+    author = re.sub(r'^([a-zäöüß]+(ed|t))? ?(by|von) ', '', author, flags=re.IGNORECASE)
```

Comment:
Yes, it's more logical to move the regexes here. It would be even better to pre-compile them above, small performance gains can be expected from moving regex compilation out of the loops.